### PR TITLE
Game rules page

### PR DESCRIPTION
**Pull request by DrimLover**
Work on the rules and a component that draws it

### CODE REVIEW

File: package.json
Comment (stelikov) [inline package.json:1]: убери эту штуку

File: package-lock.json
Comment (stelikov) [inline package-lock.json:1]: и эту бо хз что это

File: matrixscrabble.client/src/Components/GamesPage/RulesModal.tsx [hunk 1]
```diff
@@ -1,212 +1,57 @@
...
-    val.push(d);
-  });
-
-  GameWordField.push(val);
-});
+const correctWord = generateGameWordField(["   ", "rai", "   "]);
+const wrongInitialWordUsage = generateGameWordField(["ton", "   ", "   "]);
+const correctSkippedLetter = generateGameWordField(["   ", " ow", "   "]);
+const correctOutsideLetters = generateGameWordField(["   ", "   ", "utb"]);
+const scoringExample = generateGameWordField(["lic", "ow ", "utb", "kkk", " le"]);
```
Comment (akbrest) [inline matrixscrabble.client/src/Components/GamesPage/RulesModal.tsx:14]: move this variables inside component

File: matrixscrabble.client/src/Components/GamesPage/RulesModal.tsx [hunk 2]
```diff
@@ -1,212 +1,57 @@
...
 
 const RulesModal: React.FC<RulesInterface> = ({ show, handleClose }) => {
```
Comment (akbrest) [inline matrixscrabble.client/src/Components/GamesPage/RulesModal.tsx:16]: Creater new folder in Components, call it Rules and move here RulesModal

File: matrixscrabble.client/src/Components/UI/RulesExampleDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+interface RulesExampleDrawerProps {
+    matrixLetterList: string[][];
+    matrixWord: string;
+    greenLineID?: number[]; 
+    redLineID?: number[];
+    lettersOnLeft?: string[];
+    lettersOnRight?: string[];
+    lineScore?: string[];
+}
+const RulesExampleDrawer: React.FC<RulesExampleDrawerProps> = ({
```
Comment (akbrest) [inline matrixscrabble.client/src/Components/UI/RulesExampleDrawer.tsx:11]: Move RulesExampleDrawer to new folder Rules

**Review by akbrest — APPROVED** [review]

**Pull request merged**